docker/cytoscape/src/connect.sh

```diff
@@ -1,4 +1,61 @@
 #!/bin/bash
+
+if (($# == 0)); then
+    exit 0
+fi
+
+POSITIONAL_ARGS=()
+
+while [[ $# -gt 0 ]]; do
+  case $1 in
+    -s|--save)
+      SAVE="$2"
+      shift # past argument
+      shift # past value
+      ;;
+    -y|--style)
+      STYLE="$2"
+      shift # past argument
+      shift # past value
+      ;;
+    -a|--annot)
+      ANNOTATIONS="$2"
+      shift # past argument
+      shift # past value
+      ;;
+    -n|--network)
+      NETWORK="$2"
+      shift # past argument
+      shift # past value
+      ;;
+    -d|--datatypes)
+      DATATYPES="$2"
+      shift # past argument
+      shift # past value
+      ;;
+    -*|--*)
+      echo "Unknown option $1"
+      exit 1
+      ;;
+    *)
+      POSITIONAL_ARGS+=("$1") # save positional arg
+      shift # past argument
+      ;;
+  esac
+done
+set -- "${POSITIONAL_ARGS[@]}" # restore positional parameters
+
+echo "SAVE FILE         = ${SAVE}"
+echo "STYLE FILE        = ${STYLE}"
+echo "ANNOTATIONS FILE  = ${ANNOTATIONS}"
+echo "NETWORK FILE      = ${NETWORK}"
+echo "DATA TYPES        = ${DATATYPES}" 
+
+if [[ -n $1 ]]; then
+    echo "Last line of file specified as non-opt/last argument:"
```

Comment:
Since `connect.sh` is baked into the docker image, in theory there will never be a case where no arguments are specified. We should discuss the elements that we are certain will always be there when running the workflow.